spec.html

```diff
@@ -772,7 +772,7 @@ <h1>Grammar Notation</h1>
   <emu-clause id="sec-algorithm-conventions">
     <h1>Algorithm Conventions</h1>
     <p>The specification often uses a numbered list to specify steps in an algorithm. These algorithms are used to precisely specify the required semantics of ECMAScript language constructs. The algorithms are not intended to imply the use of any specific implementation technique. In practice, there may be more efficient algorithms available to implement a given feature.</p>
-    <p>Algorithms may be explicitly parameterized, in which case the names and usage of the parameters must be provided as part of the algorithm's definition.</p>
+    <p>Algorithms may be explicitly parameterized, in which case the names and usage of the parameters must be provided as part of the algorithm's definition. <dfn>Optional parameters</dfn> are denoted with surrounding brackets (`[ , _name_ ]`) and are no different from mandatory parameters within algorithm steps. A <dfn>rest parameter</dfn> may appear at the end of a parameter list, denoted with leading elipsis (`...name`). The rest parameter captures all of the arguments provided following the mandatory and optional parameters, if any, into a possibly-empty List.</p>
```

Comment:
> The rest parameter captures all of the arguments provided following the mandatory and optional parameters, if any, into a possibly-empty List.

I think it'd be good to explicitly say the parameter introduces an alias. Maybe:

> A rest parameter introduces an alias whose name is the name of the parameter and which holds a possibly-empty List whose elements are any arguments provided following the mandatory and optional parameters, in order.

---

also, s/elipsis/ellipsis/

Reply:
it's not actually an ellipsis (`…`) - it's three dots (`...`) for consistency with JS spread/rest syntax.

Reply:
@ljharb An ellipsis doesn't need to be written as the precomposed code point. See https://en.wikipedia.org/wiki/Ellipsis. It's still an ellipsis either way.

Reply:
fair enough, but i still find the term confusing since it *includes* the precomposed code point, and that won't appear in our prose.

Reply:
But I show an example of how we will write it immediately following it. And if someone searches for the precomposed code point, modern browsers will also find the decomposed results (and vice versa).

Reply:
That last point in particular (something i was unaware of) resolves my concern, thanks.